api_files/dependencies.json

```diff
@@ -60,5 +60,20 @@
                 ]
             }
         }
+    },
+    "deepLinks": {
+        "state.characters[].demographics": [
+            {
+                "condition": {
+                    "rank": "O-3",
+                    "military_branch": "US Navy"
+                },
+                "requirement": {
+                    "rank_title": [
+                        "Lieutenant"
```

Comment:
So I was testing this.  When I changed line 172 to "US Navy" I got the error started and that was good, it should error.  However when I changed line 174 to "Lieutenant" it didn't fix the error I got a different error which was:

Warn (yamlValidator):	 Key 'rank_title' at level 'demographics' must be one of the following values: ['Private (Recruit)', 'Private', 'Private First Class', 'Specialist', 'Corporal', 'Sergeant', 'Staff Sergeant', 'Sergeant First Class', 'Master Sergeant', 'First Sergeant', 'Sergeant Major', 'Command Sergeant Major', 'Sergeant Major of the Army', 'Warrant Officer 1', 'Chief Warrant Officer 2', 'Chief Warrant Officer 3', 'Chief Warrant Officer 4', 'Chief Warrant Officer 5', '2nd Lieutenant', '1st Lieutenant', 'Captain', 'Major', 'Lieutenant Colonel', 'Colonel', 'Brigadier General', 'Major General', 'Lieutenant General', 'Army Chief of Staff (special)', 'General', 'Airman Basic', 'Airman', 'Airman First Class', 'Senior Airman', 'Technical Sergeant', 'Senior Master Sergeant', 'First Sergeant / Chief Master Sergeant', 'Chief Master Sergeant of the Air Force', 'Air Force Chief of Staff (special)', 'Seaman Recruit', 'Seaman Apprentice', 'Seaman', 'Petty Officer Third Class', 'Petty Officer Second Class', 'Petty Officer First Class', 'Chief Petty Officer', 'Senior Chief Petty Officer', 'Master Chief Petty Officer', 'Master Chief Petty Officer of the Navy', 'Master Chief Petty Officer of the Coast Guard', 'Chief Warrant Officer', 'Ensign', 'Lieutenant, Junior Grade', 'Lieutenant Commander', 'Commander', 'Rear Admiral (Lower Half)', 'Rear Admiral (Upper Half)', 'Vice Admiral', 'Chief of Naval Operations (special)', 'Commandant of the Coast Guard (special)', 'Admiral', 'Lance Corporal', 'Gunnery Sergeant', 'Master Gunnery Sergeant', 'Sergeant Major of the Marine Corps', 'Warrant Officer', 'Commandant of the Marine Corps', 'Specialist 1', 'Specialist 2', 'Specialist 3', 'Specialist 4', 'Chief Master Sergeant', 'Chief Master Sergeant of the Space Force', 'Chief of Space Operations'] but is 'Lieutenant' instead.
Warn (yamlValidator):	 Key 'unstructured' at level 'Scenes/State' should be type 'string' but is <class 'NoneType'> instead.

So is my enum just not updated?  or did I do something else wrong?

Reply:
I got the same issue, but the problem is with the `api.yaml` file, not the validator or your testing.  It turns out I missed `Lieutenant` in creating all of the valid values for `rank_title`.  It's only used in Coast Guard/Navy (all other Lieutenants have some type of modifier).

Reply:
Is this an action item for me?

Reply:
Completed